### ISSUE TITLE
Error: Invalid JSON RPC response: undefined

### ISSUE THREAD

**Issue by Ashwin-Kapes** [ConsenSys-archive/ganache-cli-archive#442]
Executing a function after deploying a contract gives me an error Error: Invalid JSON RPC response: undefined However it's working fine on localhost:8000

Currently installed packages in my system.
```
F:\Blockchain\Learn\embark_demo>npm -g list --depth=0
C:\Users\HP\AppData\Roaming\npm
+-- embark@2.6.0
-- ethereumjs-testrpc@6.0.3
```
Later I uninstalled ethereumjs-testrpc and installed ganache-cli but getting same error.
Here's the snapshot:
![delete_embark](https://user-images.githubusercontent.com/510118/33683909-7865dbda-daf2-11e7-8ef0-1fbd95bb77f5.png)

**Comment by benjamincburns** [comment]
@Ashwin-Kapes sorry for the late response on this!

I've seen this issue while using Truffle as well, but it's intermittent and I haven't been able to nail it down. By any chance does this reproduce reliably for you? If so, could you possibly package up that reproduction somehow? A PR w/ a test that fails would be amazing, but I'm totally happy with some easy-to-follow numbered steps submitted via a comment here.

**Comment by benjamincburns** [comment]
I've just published `ganache-cli@7.0.0-beta.0` which I believe fixes this bug, as well as adds support for websockets, and contains numerous other fixes. You can get it by doing `npm install -g ganache-cli@beta`.

Closing this case for now, but please reopen if I've got it wrong and this problem shows its ugly face again.
